.circleci/config.yml

```diff
@@ -544,6 +544,10 @@ jobs:
       - tag-push-docker:
           project: orc8r
           images: "nginx|controller"
+      - tag-push-docker:
+          project: orc8r
+          images: "nginx|controller"
+          tag: "latest"
```

Comment:
Let's use `tag-latest: true`

Cause if we do `tag: "latest"` it will only tag the binaries with latest and not the commit hash, if we pass the above parameter it will double tag it with `latest` and `commit hash`